config/prod/tom-test-ubuntu16.yaml

```diff
@@ -0,0 +1,47 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
+stack_name: tom-test-ubuntu16
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "ami-08765b6c8def4c8bb"    # Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-02-12
```

Comment:
sorry this AMI is actually Mike's AMI from his personal account.  We shared it with sandbox but its is ubuntu 16.04.